### PR TITLE
remove countryStatus, use country.props

### LINKED ISSUE #1226: Remove CountryStatus
CountryStatus migrated to Country.props

CountryStatus left as helper:

- Remove references to CountryStatus (ex. store actions, backend repository updates)
- Replace above with country.props

### PR DESCRIPTION
**Pull request by sorja**
Resolves #1226

### CODE REVIEW

File: client/store/user/hooks/index.ts [hunk 1]
```diff
@@ -2,8 +2,9 @@ import { useCountryIso } from '@client/hooks'
 import { CountryIso } from '@meta/area'
 import { Authorizer, User, Users } from '@meta/user'
 import { useAppSelector } from '@client/store'
-import { useAssessmentCountryStatus, useAssessmentSection, useCountries } from '@client/store/assessment'
+import { useAssessmentSection, useCountries } from '@client/store/assessment'
 import { useIsDataLocked } from '@client/store/ui/dataLock'
+import { useAssessmentCountry } from '@client/store/assessment/hooks'
```
Comment (minotogna) [inline client/store/user/hooks/index.ts:7]: better to export useAssessmentCountry from '@client/store/assessment'

File: server/repository/assessment/updateCountry.ts [hunk 2]
```diff
@@ -14,12 +14,12 @@ export const updateCountry = async (
...
     `
         update ${assessmentCycleName}.country
-        set props = props || $2
+        set props = $2
         where country_iso = $1
         returning *;
 
     `,
-    [countryIso, JSON.stringify(country.props)],
+    [countryIso, country.props],
```
Comment (minotogna) [inline server/repository/assessment/updateCountry.ts:22]: no stringify needed?

**Review by minotogna — COMMENTED** [review]
👍

File: server/repository/assessment/updateCountry.ts (hunk 2 above)
Comment (sorja) [reply server/repository/assessment/updateCountry.ts:22]: works without

Comment (sorja) [reply server/repository/assessment/updateCountry.ts:22]: regarding this i wrote in dev:

how............... can this be possible?
https://github.com/openforis/fra-platform/pull/1239/files#diff-6e90bef548c0033f2e457073433be4dbfdbcc2c5858af6a0e04ce397a3a850dbR17
[7:02](https://fraplatform.slack.com/archives/C565LS99U/p1649433734517249)
This only works if i dont concat the object ( || $2 )
[7:04](https://fraplatform.slack.com/archives/C565LS99U/p1649433854850279)
If I concat:

    Updating country.props.status from editing to review works
    Updating country.props.desk_study from false to true does not work....?

**Review by minotogna — APPROVED** [review]

**Pull request merged**